Multithreading2/src/main/java/org/hillel/Multithreading2/ThreadSafeList/ThreadSafeList.java

```diff
@@ -0,0 +1,27 @@
+package org.hillel.Multithreading2.ThreadSafeList;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ThreadSafeList<E> {
+  private final List<E> list = Collections.synchronizedList(new ArrayList<>());
```

Comment:
here you dont need convert to Array list to synchronised one, your methods will handle it 

so simply `private final List<E> list = new ArrayList<>();`